frontend/src/common/enums/api/api-path.enum.ts

```diff
@@ -1,5 +1,6 @@
 enum ApiPath {
   AUTH = '/auth',
+  REFRESH_TOKEN = '/refreshToken',
```

Comment:
These paths should include the full URL, so the `REFRESH_TOKEN` should be:
```suggestion
  AUTH = '/auth',
  REFRESH_TOKEN = '/auth/refreshToken',
```